kernel/inferCumulativity.ml

```diff
@@ -172,9 +170,27 @@ let rec infer_fterm cv_pb infos variances hd stk =
   | FRel _ -> infer_stack infos variances stk
   | FInt _ -> infer_stack infos variances stk
   | FFloat _ -> infer_stack infos variances stk
-  | FFlex fl ->
-    let variances = infer_table_key (info_env (fst infos)) variances fl in
-    infer_stack infos variances stk
+  | FFlex Names.(RelKey _ | VarKey _ as fl) ->
+    (* We could try to lazily unfold but then we have to analyse the
+       universes in the bodies, not worth coding at least for now. *)
+    begin match unfold_ref_with_args (fst infos) (snd infos) fl stk with
+    | Some (hd,stk) -> infer_fterm cv_pb infos variances hd stk
+    | None -> infer_stack infos variances stk
+    end
+  | FFlex (Names.ConstKey con as fl) -> begin
+      let def = unfold_ref_with_args (fst infos) (snd infos) fl stk in
```

Comment:
Nitpick: the indentation is not consistent with the other branches.

Reply:
(I.e. move the begin on its own line.)

Reply:
Moving the begin doesn't change indentation.
Do you want me to indent less than the automation suggests? I can do that.

Reply:
No, I mean leave the block alone and just move the begin on its own line with less indentation than the inner block.

Reply:
I don't see the point.

Reply:
The current code slightly confused me because I didn't match the end block with it's opening begin immediately.